libs/dh/market-participant/data-access-api/src/lib/dh-market-participant-edit-actor-data-access-api.store.ts

```diff
@@ -84,6 +81,12 @@ export class DhMarketParticipantEditActorDataAccessApiStore extends ComponentSto
   actor$ = this.select((state) => state.actor);
   validation$ = this.select((state) => state.validation);
   changes$ = this.select((state) => state.changes);
+  marketRoles$ = this.select((state) => state.marketRoles);
```

Comment:
suggestion: `marketRoles$` no need to be exposed. It's not used anywhere later on. I think I added it so I can test locally.